pkg/northbound/gnmi/subscribe_test.go

```diff
@@ -172,35 +141,36 @@ func Test_SubscribeLeafStream(t *testing.T) {
 
 	//FIXME Waiting for subscribe to finish properly --> when event is issued assuring state consistency we can remove
 	time.Sleep(100000)
+
+	var deletePaths = make([]*gnmi.Path, 0)
+	var replacedPaths = make([]*gnmi.Update, 0)
+	var updatedPaths = make([]*gnmi.Update, 0)
+	//augmenting pre-existing value by one
+	typedValue := gnmi.TypedValue_StringVal{StringVal: "14"}
+	value := gnmi.TypedValue{Value: &typedValue}
+	updatedPaths = append(updatedPaths, &gnmi.Update{Path: path, Val: &value})
+	setRequest := &gnmi.SetRequest{
+		Delete:  deletePaths,
+		Replace: replacedPaths,
+		Update:  updatedPaths,
+	}
+	//Sending set request
 	go func() {
-		_, err = server.Set(context.Background(), &setRequest)
+		_, err = server.Set(context.Background(), setRequest)
```

Comment:
again the err never gets used here because it's on a separate thread

Reply:
done